composer.json

```diff
@@ -116,6 +116,18 @@
                 }
             }
         },
+        {
```

Comment:
@NightHunterSV please create a PR with this change to https://github.com/ymcatwincities/openy-project/blob/8.2.x-development/composer.json

Reply:
OK, got this